Livecheckables/speech-tools.rb

```diff
@@ -1,4 +1,4 @@
 class SpeechTools
-  livecheck :url   => "http://festvox.org/packed/festival/",
-            :regex => %r{href="([0-9\.]+)/">}
+  livecheck :url   => "http://festvox.org/packed/festival/?C=M&O=D",
+            :regex => %r{href="(\d+(?:\.\d+)+)/">}
```

Comment:
```suggestion
            :regex => %r{href="(\d+(?:\.\d+)+)/?"}
```

The ending `"` is doing the work of restricting the regex to the versions we want, so we don't need the trailing `>` here. We can also make this more general by making the trailing `/` optional (i.e., `/?`). There's always a chance that something weird can happen upstream and maybe a version link won't have the trailing `/`, so it's nice to allow for the possibility when it doesn't detract from the regex's ability to properly match what we want.